UIViewExtension.swift

```diff
@@ -49,5 +49,10 @@ extension UIView {
 		get{ return self.frame.size }
 		set(newValue){ self.frame.size = newValue }
 	}
+    
+    // rounding the corners of the view's layer.
+    func roundCorners(radius: CGFloat = CGFloat(10)) {
```

Comment:
Can remove the explicit cast to `CGFloat(10` in there because of the explicit typing of the parameter. Just putting it in as `10` will infer that it's a `CGFloat`